templates/django.conf

```diff
@@ -71,6 +71,7 @@ server {
     location @django {
         uwsgi_pass  {{ django_name }};
         include     uwsgi_params;
+        client_max_body_size: {{ nginx_client_max_body_size }};
```

Comment:
No `:` character after the key's name :)

Reply:
So true.

See 236284c